rclpy/test/test_logging.py

```diff
@@ -359,6 +359,22 @@ def test_clear_config(self):
             rclpy.logging._root_logger.get_effective_level(),
             my_logger.get_effective_level())
 
+    def test_logging_severity_from_string(self):
+        log_levels = {
+            'UNSET': 0,
+            'DEBUG': 10,
+            'INFO': 20,
+            'WARN': 30,
+            'ERROR': 40,
+            'FATAL': 50
+        }
```

Comment:
@BMarchi hmm should we hard-code these integers here? Doesn't it defeat the purpose?

Reply:
Why? Having them outside the test doesn't make any sense. I would even said that having an rcl function to make that won't have sense at all if that was the case.

Reply:
Actually, looking into the [`rclpy.logging` module](https://github.com/ros2/rclpy/blob/f293718b235ae3e86b2b4518ef0e9d2da3f2a2e9/rclpy/rclpy/logging.py#L22) I see severity levels are already duplicated w.r.t. [`rcutils`](https://github.com/ros2/rcutils/blob/fe215b4a3db580358f09bda6269e1356165fa920/include/rcutils/logging.h#L164).

I don't really like either, as it couples the implementation with that of a transitive dependency. But that's another discussion. Let's just use what's already defined in `rclpy.logging` i.e. let's keep the duplication in a single place.

Reply:
> Actually, looking into the rclpy.logging module I see severity levels are already duplicated w.r.t. rcutils.

Pointing out one location where code was copies hardly justifies to repeat the same anti pattern. Instead it motivates to the remove the existing duplication instead.

Reply:
I'm not arguing for duplication, we should use the Python `enum` that's available and will likely stay available, regardless of how we manage to construct it. We can then deal with how we tunnel this information without duplication in a follow up PR. Do you agree @dirk-thomas ?

Reply:
> We can then deal with how we tunnel this information without duplication in a follow up PR

Sure (as long as the follow up PR is happening in the near future).